src/main/java/me/iblitzkriegi/vixio/expressions/message/ExprEmojis.java

```diff
@@ -20,41 +20,37 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ExprEmojis extends SimpleExpression<Emoji> {
+public class ExprEmojis extends ChangeableSimpleExpression<Emoji> implements EasyMultiple<Message, Emoji> {
+
     static {
-        Vixio.getInstance().registerExpression(ExprEmojis.class, Emoji.class, ExpressionType.SIMPLE,
-                "[the] (emoji|emote|reaction)[s] of %message% [(with|as) %-bot/string%]")
+        Vixio.getInstance().registerPropertyExpression(ExprEmojis.class, Emoji.class,
+                "reactions", "messages")
                 .setName("Reactions of message")
-                .setDesc("Get the emojis of a message. Changers: DELETE, RESET, REMOVE ALL, REMOVE, ADD")
+                .setDesc("Get the reactions of a message. Can be deleted, reset, removed and added to.")
```

Comment:
Need to include that a bot is required to modify them.

Reply:
i recommend you append this to the description automatically when you rework the docs system